source/user-guide/creating-resources.html.md.erb

```diff
@@ -36,5 +36,7 @@ These resources are preconfigured with a backend, providers etc, all that you ne
 
 - The ability to create some resources will be restricted and will require assistance from the Modernisation Platform team.
 
+- There are folders in place which start with "platform_". Changes should not be made to these without consulting the Modernisation Platform team. Changes to these folders may be blocked.
```

Comment:
Did you mean `files` here, rather than `folders`?

Reply:
New issue pushed